reference/7.4/Microsoft.PowerShell.Core/About/about_Preference_Variables.md

```diff
@@ -1122,11 +1122,25 @@ errors according to `$ErrorActionPreference` when
 > `PSNativeCommandUseErrorActionPreference` is an experimental feature added in
 > PowerShell 7.3. For more information, see [Using experimental features][13].
 
+Some native commands, like [robocopy][14] use non-zero exit codes to represent
+information other than errors. In these cases, you can temporarily disable the
+behavior and prevent non-zero exit codes from issuing errors.
+
+```powershell
+$PSNativeCommandUseErrorActionPreference = $false
```

Comment:
Should we save the current setting then restore it?

Reply:
Oh, that would be better. Will do!